Podcast/EmptyStateView.swift

```diff
@@ -79,8 +79,8 @@ enum EmptyStateType {
         switch self {
         case .pastSearch:
             return #imageLiteral(resourceName: "searchIcon")
-        case .bookmarks:
-            return #imageLiteral(resourceName: "bookmark_empty_state")
+//        case .bookmarks:
+//            return #imageLiteral(resourceName: "bookmark_empty_state")
```

Comment:
Why commented?

Reply:
in case we want to use this image again, I don't want us to lose the image. It looks bad though with jump back in and the image so that's why its no longer showed.